app/components/Sidebar.tsx

```diff
@@ -55,16 +67,113 @@ const JumpToButton = () => {
 }
 
 export function Sidebar({ children }: { children: React.ReactNode }) {
+  const AnimatedDialogContent = animated(Dialog.Content)
+  const { isOpen } = useMenuState()
+  const config = { tension: 1200, mass: 0.125 }
+  const { pathname } = useLocation()
+
+  const transitions = useTransition(isOpen, {
+    from: { x: -50 },
+    enter: { x: 0 },
+    config: isOpen ? config : { duration: 0 },
+  })
+
   return (
-    <div className="flex flex-col border-r text-sans-md text-default border-secondary">
-      <div className="mx-3 mt-4">
-        <JumpToButton />
-      </div>
-      {children}
+    <>
+      {transitions(
+        ({ x }, item) =>
+          item && (
+            <Dialog.Root
+              open
+              onOpenChange={(open) => {
+                if (!open) closeSidebar()
+              }}
+              // https://github.com/radix-ui/primitives/issues/1159#issuecomment-1559813266
+              modal={false}
+            >
+              <div
+                aria-hidden
+                className="fixed inset-0 top-[61px] z-10 overflow-auto bg-scrim lg+:hidden"
+              />
+              <AnimatedDialogContent
+                className="fixed z-sideModal flex h-full w-[14.25rem] flex-col border-r text-sans-md text-default border-secondary lg+:!transform-none lg-:inset-y-0 lg-:top-[61px] lg-:bg-default lg-:elevation-2"
+                style={{
+                  transform: x.to((value) => `translate3d(${value}%, 0px, 0px)`),
+                }}
+                forceMount
+              >
+                <div className="mx-3 mt-4">
+                  <JumpToButton />
+                </div>
+                {children}
+                {pathname.split('/')[1] !== 'settings' && <ProfileLinks />}
+              </AnimatedDialogContent>
+            </Dialog.Root>
+          )
+      )}
+    </>
+  )
+}
+
+export const ProfileLinks = () => {
```

Comment:
Adds profile info into the navigation, hidden on desktop because it's visible in the nav.